SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -2906,13 +2843,130 @@ void CNEMOEulerSolver::BC_Supersonic_Outlet(CGeometry *geometry, CSolver **solut
 
 }
 
-//void CNEMOEulerSolver::BC_Sym_Plane(CGeometry *geometry, CSolver **solver_container,
-//                                    CNumerics *conv_numerics, CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
-//
-//  /*--- Call the Euler wall routine ---*/
-//  BC_Euler_Wall(geometry, solver_container, conv_numerics, visc_numerics, config, val_marker);
-//
-//}
+void CNEMOEulerSolver::BC_Euler_Wall(CGeometry *geometry, CSolver **solver_container,
+                                    CNumerics *conv_numerics, CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
+
+  unsigned short iDim, jDim, iSpecies, iVar, jVar;
+  unsigned long iPoint, iVertex;
+
+  su2double *Normal = nullptr, Area, UnitNormal[3], *NormalArea,
+  **Jacobian_b, **DubDu,
+  rho, cs, P, rhoE, rhoEve, conc, *u, *dPdU;
+
+  bool implicit = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
+
+  /*--- Allocate arrays ---*/
+  Normal     = new su2double[nDim];
+  NormalArea = new su2double[nDim];
+  Jacobian_b = new su2double*[nVar];
+  DubDu      = new su2double*[nVar];
+  u          = new su2double[nDim];
+
+  for (iVar = 0; iVar < nVar; iVar++) {
+    Jacobian_b[iVar] = new su2double[nVar];
+    DubDu[iVar] = new su2double[nVar];
+  }
```

Comment:
You can initialize to zero by 
for (iVar = 0; iVar < nVar; iVar++) {
  Jacobian_b[iVar] = new su2double[nVar] **()**;